tracer/src/Datadog.Trace/ClrProfiler/TracerDebugger.cs

```diff
@@ -40,7 +40,7 @@ private static bool WaitForManagedDebugger()
         }
 
         var debugEnabled = EnvironmentHelpers.GetEnvironmentVariable(ConfigurationKeys.WaitForDebuggerAttach);
-        if (!string.IsNullOrEmpty(debugEnabled) && debugEnabled.Equals("1", StringComparison.Ordinal))
+        if (!string.IsNullOrEmpty(debugEnabled) && debugEnabled?.Equals("1", StringComparison.Ordinal) == true)
```

Comment:
I think you should use `!` here, we know that `debugEnabled` is not null as per the `string.IsNullOrEmpty` check

Reply:
👍 Stupid .NET FX grmbl grmbl